usr/src/test/zfs-tests/cmd/scripts/zfstest.ksh

```diff
@@ -72,10 +72,6 @@ function verify_id
 
 	sudo -n id >/dev/null 2>&1
```

Comment:
While here, could we change sudo invocation to include '-k' to ignore the timestamp - I was bitten several times by this, where I would use sudo with a password and then run the suite within the timeout, so the check passes, but after timeout is expired, the entire run fails.